### ISSUE TITLE
show expressivity and penetrance in term detail view

### ISSUE THREAD

**Issue by ValWood** [pombase/website#308]
did we used to? I'm sure that we did
http://pombase2.aska.gen.nz/term/FYPO%3A0000099

**Comment by mah11** [comment]
Term detail view does show expressivity and penetrance ... when any are annotated.

FYPO:0000099 will never have any penetrance extensions because it's a population term, and happens not to have any annotations with expressivity extensions.

**Comment by ValWood** [comment]
Ah OK, I thought that was odd as there are double mutants  with no expressivity, presumably much sicker than the contributing, single mutants which are also listed.... but no worries...

**Comment by mah11** [comment]
I wouldn't assume that - the comparison is always to wild type

**Comment by ValWood** [comment]
Its good to add the expressivity  to the doubles, to indicate whether the expressivity is the same or different from the single. If the phenotype if  its the same as the single,  it isn't additive (so not net effect).  It's unusual for  people show sensitivity for doubles and exclude single mutant? The same expressivity of phenotype might indicate same pathway, but it could also be that one allele might be making no contribution to the phenotype....

 I guess this indicates why double mutants aren't so useful in lists as single mutants when you  always know that there is an effect...

**Comment by mah11** [comment]
> It's good to add the expressivity to the doubles, to indicate whether the expressivity is the same or different from the single.

I agree, but it's not always practical because we don't have any absolute quantitative measure for expressivity. If an experiment just shows that yfg1 and the yfg1 yfg2 double mutant are equally sensitive to X, there's no straightforward way to say whether that expressivity is low, medium, or high.

I have made some annotations where I've included a "same as yfg1 alone" sort of comment.
